airflow/cli/commands/dag_command.py

```diff
@@ -67,6 +67,13 @@ def dag_backfill(args, dag=None):
 
     signal.signal(signal.SIGTERM, sigint_handler)
 
+    import warnings
+    warnings.warn('--ignore_first_depends_on_past is deprecated as the value is always set to True',
```

Comment:
Should be  `--ignore-first-depends-on-past` due to https://github.com/apache/airflow/pull/7148

Reply:
will submit new PR to correct it.

Reply:
thanks.